src/imageio/format/pdf.c

```diff
@@ -583,7 +583,7 @@ void gui_init(dt_imageio_module_format_t *self)
   gtk_grid_attach(grid, widget, 0, ++line, 1, 1);
 
   d->title = GTK_ENTRY(gtk_entry_new());
-  gtk_entry_set_placeholder_text(d->title, "untitled");
+  gtk_entry_set_placeholder_text(d->title, C_("pdf placeholder", "untitled"));
```

Comment:
"pdf placeholder" -> is that a name? a title? maybe just "name" or "title"?

Reply:
Frankly I'm not sure anymore why I wanted to add context here, as this is the only occurrence of "untitled"... I'll undo this one. Sorry.